sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -137,6 +139,22 @@ class JoinSuite extends QueryTest with SharedSQLContext {
     assert(planned.size === 1)
   }
 
+  test("broadcasthint sets relation statistics to smallest value") {
```

Comment:
yeah, I just can't find appropriate suite for it. I'd use JoinOptimizationSuite. Thanks.